src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/FlagsImageHelper.java

```diff
@@ -35,7 +35,7 @@ public class FlagsImageHelper {
     private static final String COLOR_WHITE = "White";
     private static final String COLOR_DEEP_PINK = "DeepPink";
     private static final String COLOR_SLATE_GRAY = "SlateGray";
-    private static final String COLOR_YELLOW = "Yellow";
+    private static final String COLOR_DARK_BLUE = "DarkBlue";
```

Comment:
RestrictedReporting - if you select Yes for Rule50(3)(d) - even thought the colour is LightBlack - it appears almost like a dark blue to my eyes anyway. It might be worth checking to see if they are different. But like I said, it what my eyes see.